batch10/assignment4/README.md

```diff
@@ -0,0 +1,11 @@
+# Assignment 4
+
+Tasks to be completed by next class
+
+- Complete all the labs >> Done
+- Try to understand what the commands are doing >> practice in progress
+- Describe Architecture of Kubernetes and push to Github repo 
+    >> On High Level there are Master Controller and Slave Controllers.
```

Comment:
Its not controller rather master & worker nodes...
Controllers are totally different components running on master nodes